templates/taxi/manufacturer_confirm_delete.html

```diff
@@ -0,0 +1,11 @@
+{% extends "base.html" %}
+
+{% block content %}
+  <h1>Deleting Car Manufacturer</h1>
+  <p>You are going to delete <strong>{{ manufacturer.name }}</strong> manufacturer.</p>
+  <p><i>This delete will lead you to losing cars with this manufacturer in the taxi service</i></p>
+  <form action="" method="post">
+    {% csrf_token %}
+    <input type="submit" class="btn btn-danger">

```

Comment:
Consider adding a value to the submit button, such as `value="Yes, delete the manufacturer"`, to make the action clearer to the user.